src/driver/postgres/PostgresDriver.ts

```diff
@@ -370,7 +384,7 @@ export class PostgresDriver implements Driver {
             || columnMetadata.type === "timestamp without time zone") {
             return DateUtils.mixedDateToDate(value);
 
-        } else if (columnMetadata.type === "json" || columnMetadata.type === "jsonb") {
+        } else if (this.spatialTypes.concat(["json", "jsonb"]).indexOf(columnMetadata.type) >= 0) {
```

Comment:
previous one was better =)

Reply:
_waits for `Array.includes` in TS..._

Local style should be:

```typescript
} else if (columnMetadata.type === "json" ||
           columnMetadata.type === "jsonb" ||
           this.spatialTypes.indexOf(columnMetadata.type) >= 0) {
```

❓

Reply:
no, thats fine as well, you can keep it. I simply don't like `concat`, I would either go with "local style" you wrote or `["json", "jsob", ... this.spatialTypes].indexOf()` which looks much better then concat

Reply:
Ah, right, I always forget about spreads in this context.